### ISSUE TITLE
Is it possible to add partial deletion for singular species in dsmcDeletionPatch?

### ISSUE THREAD

**Issue by PrikMushi** [vincentcasseau/hyStrath#135]
I am working on a project where I want the dsmcDeletionPatch to delete particles, but at the same time reflect some particles backwards. Is it possible to add some probability here, where some particles hitting the deletion patch do not get detelted but get reflected?

I have only one gas species in the study.

**Comment by timteichmann** [comment]
Take a look at `dsmcAbsorbingWallPatch` and associated mixed boundaries, e.g. `dsmcAbsorbingDiffuseWallPatch`.

**Comment by PrikMushi** [comment]
Thank you.

From my understanding, absorbing doesn't equate to deletion from the domain? I would still like to implement something sort of deletion patch plus some particles being reflected back.

**Comment by timteichmann** [comment]
> From my understanding, absorbing doesn't equate to deletion from the domain?

Yes, it does. For more details please have a look at the source code.

**Comment by PrikMushi** [comment]
Thank you again.

 I have never used the Absorption patch. Is the syntax for absorption probability as such? For 50% absorption of Argon.
 
absorptionProbabilities = 50_[Ar];

**Comment by du-zhihua** [comment]
How to use this boundary condition? I just need to use dsmcAbsorbingDiffuseWallPatch.
